src/registry.rs

```diff
@@ -53,11 +54,11 @@ pub trait IntoPortable {
     fn into_portable(self, registry: &mut Registry) -> Self::Output;
 }
 
-impl IntoPortable for &'static str {
+impl IntoPortable for <MetaForm as Form>::String {
```

Comment:
Can this be restricted to `String` only? This conflicts with all the other `IntoPortable` impls for the specs, but works fine when I change it to  `String`.

Reply:
I was able to remove this completely :point_down:

Reply:
That helps